frontend/src/__tests__/cypress/cypress/utils/oc_commands/dataConnection.ts

```diff
@@ -0,0 +1,28 @@
+import { replacePlaceholdersInYaml } from '~/__tests__/cypress/cypress/utils/yaml_files';
+import { applyOpenShiftYaml } from './baseCommands';
+
+/**
+ * Try to create a data connection based on the dataConnectionReplacements config
+ * @param dataConnectionReplacements Dictionary with the config values
+ *      Dict Structure:
+ *              dataConnectionReplacements = {
+ *                  NAMESPACE: <PROJECT NAME>,
+ *                  AWS_ACCESS_KEY_ID: <AWS ACCESS KEY ID>,
+ *                  AWS_DEFAULT_REGION: <AWS REGION>,
+ *                  AWS_S3_BUCKET: <AWS BUCKET NAME>,
+ *                  AWS_S3_ENDPOINT: <AWS ENDPOINT>,
+ *                  AWS_SECRET_ACCESS_KEY: <AWS SECRET>,
+ *               }
+ * @param yamlFilePath
+ */
+export const createDataConnection = (
+  dataConnectionReplacements: { [key: string]: string },
```

Comment:
Add proper types.

Reply:
Fixed in [caef85f](https://github.com/opendatahub-io/odh-dashboard/pull/3028/commits/caef85f876c45971b62538a3c03ca5f91e3a3627)